packages/utils/src/modify-config-helper.ts

```diff
@@ -133,13 +133,13 @@ export function modifyHelperUtil(
                 process.exitCode = -1;
             }
 
-            const transformConfig: TransformConfig = Object.assign(
+            const transformConfig = Object.assign(
```

Comment:
The `TransformConfig` type and the `WebpackScaffoldObject` type both had the field `config`, but both config fields were different. When the object.assign kicked in, it changed the type to `TransformConfig & WebpackScaffoldObject`, but was then trying to put it into a `TransformConfig`. Which then complained that the ScaffoldObject one didnt have a required merge field. So without understanding these types fully, ive opted to explicitly cast the `transformConfig` as the type it originally was. Perhaps someone can give me more light what these different types mean, and if the `config` field can be shared perhaps, or handle the case typescript is highlighting?